src/main/java/com/synapps/moj/dfs/response/CourtLog.java

```diff
@@ -44,4 +46,10 @@ public void setCaseNumber(String value) {
         this.caseNumber = value;
     }
 
+    public void addCourtLogEntry(CourtLogEntry courtLogEntry) {
+        if(isNull(entry)) {
+            entry = new ArrayList<>();
+        }
+        entry.add(courtLogEntry);
```

Comment:
Since `getEntry()` already does some of this work, consider simplifying to:
```java
    public void addCourtLogEntry(CourtLogEntry courtLogEntry) {
        getEntry().add(courtLogEntry);
    }
```